protocol/error.go

```diff
@@ -26,8 +34,11 @@ const (
 	ErrorCouldNotVerify
 )
 
-// ErrorResponses contains error codes that
-// a response can omit the DirectoryResponse.
+// ErrorResponses contains the server responses indicating that
+// a client request could not be processed due to a
+// malformed client request or an internal server error.
+// Server response messages containing one of these errors therefore
+// omit the DirectoryResponse.
```

Comment:
> we can keep Error- prefix for error codes in ErrorResponses.

Yes, definitely, sorry if this wasn't clear. My idea was to distinguish between actual errors and results.